lib/tramway/admin/generators/install_generator.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails/generators'
+require 'tramway/admin/generators/model_generator'
+
+module Tramway
+  module Admin
+    module Generators
+      class InstallGenerator < ::Rails::Generators::Base
+        source_root File.expand_path('templates', __dir__)
+
+        def run_decorator_generators
+          ::Tramway::Admin.available_models_for(project).map do |model|
+            generate 'tramway:admin:model', model.to_s
```

Comment:
`Tramway::User::User.role.values`

Reply:
"tramway:#{current_user.role}:model"

Reply:
fixed with argument for generator(see Readme)